src/content-helper/dashboard-widget/class-dashboard-widget.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Parse.ly Content Helper Dashboard Widget class
+ *
+ * @package Parsely
+ * @since   3.7.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\ContentHelper;
+
+use Parsely\Parsely;
+
+use const Parsely\PARSELY_FILE;
+
+/**
+ * Class that generates and manages the Content Helper Dashboard Widget.
+ *
+ * @since 3.7.0
+ */
+class Dashboard_Widget {
+
+	/**
+	 * Sets all the hooks that are needed in order to add the Dashboard Widget
+	 * to the WordPress Dashboard.
+	 *
+	 * @since 3.7.0
+	 */
+	public function run(): void {
+		add_action( 'wp_dashboard_setup', array( $this, 'add_dashboard_widget' ) );
+		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_assets' ) );
+	}
+
+	/**
+	 * Adds the Widget and its contents to the WordPress Dashboard.
+	 *
+	 * @since 3.7.0
+	 */
+	public function add_dashboard_widget(): void {
+		wp_add_dashboard_widget(
+			'wp-parsely-dashboard-widget',
```

Comment:
We actually want this name to be generic, as we should be adding more data to it in next iterations. The reason we've only added top posts so far is because this is the only thing we had available from the API until now.

Reply:
Maybe we should change it in future then when its actually needed ...... just a suggestion

Reply:
At this point the plugin has only one Dashboard Widget, which should be populated with more data in the next iterations. As such, I doubt we get value in renaming it now and then having to rename it back.